src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/.template.config/ide.host.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "order": 660,
+  "icon": "icon.png",
+  "ports": [
```

Comment:
port section is unneeded, and should be removed. VS 2022 no longer references this section.